package.json

```diff
@@ -122,10 +123,12 @@
     "typescript": "4.3.5"
   },
   "dependencies": {
+    "@alipay/cal-exp": "^0.0.1",
     "@rollup/plugin-commonjs": "^20.0.0",
     "@rollup/plugin-node-resolve": "^13.0.4",
     "@rollup/plugin-typescript": "^8.2.5",
     "date-fns": "^2.23.0",
+    "expr-eval": "^2.0.2",
```

Comment:
If this pkg is noly for LI, it does not need to be added to the root directory

Reply:
got it.